django_project/base/models/custom_domain.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+import os
+
+
+ROLE = (
+    ('project', 'project'),
+    ('organisation', 'organisation'),
+)
+
+
+class CustomDomain(models.Model):
```

Comment:
I think we should just call it 'Domain' and then create a Domain entry for e.g. http://projecta.kartoza.com which is the default domain that all new projects get added to that do not have an explicitly set domain. Then add a foreign key reference to Domain from both the organisation and project models.